Hmmm. Looks like that test was not fully fixed for the changes in master;

```

[2020-07-16T14:37:39.495Z] _________ CreateContainerTest.test_invalid_log_driver_raises_exception _________
[2020-07-16T14:37:39.495Z] tests/integration/api_container_test.py:293: in test_invalid_log_driver_raises_exception
[2020-07-16T14:37:39.495Z]     assert excinfo.value.explanation in expected_msgs
[2020-07-16T14:37:39.495Z] E   AssertionError: assert 'error looking up logging plugin asdf: plugin "asdf" not found' in ["logger: no log driver named 'asdf' is registered", 'looking up logging plugin asdf: plugin "asdf" not found']
[2020-07-16T14:37:39.495Z] E    +  where 'error looking up logging plugin asdf: plugin "asdf" not found' = APIError(HTTPError('400 Client Error: Bad Request for url: http+docker://localhost/v1.39/containers/create')).explanation
[2020-07-16T14:37:39.495Z] E    +    where APIError(HTTPError('400 Client Error: Bad Request for url: http+docker://localhost/v1.39/containers/create')) = <ExceptionInfo APIError tblen=6>.value
```

--- a/hack/make/test-docker-py
+++ b/hack/make/test-docker-py
@@ -7,14 +7,14 @@ source hack/make/.integration-test-helpers
 # TODO docker 17.06 cli client used in CI fails to build using a sha;
 # unable to prepare context: unable to 'git clone' to temporary context directory: error fetching: error: no such remote ref ead0bb9e08c13dd3d1712759491eee06bf5a5602
 #: exit status 128
-: "${DOCKER_PY_COMMIT:=4.2.1}"
+: "${DOCKER_PY_COMMIT:=4.2.2}"
 
 # Override the API version used to make sure all tests are run
 # TODO remove this or unset this after https://github.com/docker/docker-py/pull/2512 is merged
 : "${DOCKER_PY_TEST_API_VERSION:=1.39}"
 
 # custom options to pass py.test
-# TODO remove these skip once we update to a docker-py version that has https://github.com/docker/docker-py/pull/2549: CreateContainerTest::test_invalid_log_driver_raises_exception
+# TODO remove these skip once we update to a docker-py version that has https://github.com/docker/docker-py/pull/2624: CreateContainerTest::test_invalid_log_driver_raises_exception
 #
 # This option can be used to temporarily skip flaky tests (using the `--deselect`
 # flag) until they are fixed upstream. For example:

